packaging/rpm/crc.spec.in

```diff
@@ -0,0 +1,226 @@
+# https://github.com/code-ready/admin-helper
+%global goipath         github.com/code-ready/crc
+Version:                __VERSION__
+%global 	 	openshift_suffix -__OPENSHIFT_VERSION__
+
+%gometa
+
+# debuginfo is not supported on RHEL with Go packages
+%global debug_package %{nil}
+%global _enable_debug_package 0
+
+%global common_description %{expand:
+CodeReady Container's executable}
+
+
+%global golicenses    LICENSE
+%global godocs        *.md
+
+
+Name:           %{goname}
+Release:        1%{?dist}
+Summary:        CodeReady Container's executable
+License:        APL 2.0
+ExcludeArch:    armv7hl i686
+URL:            %{gourl}
+ExcludeArch:    s390x
+Source0:        %{gosource}
+
+#generate_buildrequires
+#go_generate_buildrequires
+
+BuildRequires: git-core
+BuildRequires: go-srpm-macros
+BuildRequires: make
+
+Provides: bundled(golang(github.com/Azure/go-ansiterm)) = 0.0.0-0.20170929gitd6e3b3328b78
```

Comment:
Is it possible to automate this ? The next time we will touch go.mod I guess we will have to regenerate this.

Reply:
Need to add a rule similar to the golangci-lint one to build https://github.com/cfergeau/gomod2rpmdeps/ when needed, and then this can be automated yes. Did not get to it yet, but I  fully agree it's going to be painful to maintain if it's not automated.

Reply:
I've added this in the latest iteration